account-service/backend/app.py

```diff
@@ -0,0 +1,45 @@
+from flask import Flask, request, jsonify, session
+from flask_cors import CORS
+from login import authenticate_user
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = '3f9a4c5b6d7e8f9a0b1c2d3e4f5g6h7i8j9k0l1m2n3o4p5q6r7s8t9u0v1w2x3y4z5a6b7c8d9e0f1g2h3i4j5k6l7m8n9o0p1q2r3s4t5u6v7w8x9y0z1a2b3c4d5e6f7g8h9i0j1k2l3m4n5o6p7q8r9s0t1u2v3w4x5y6z7a8b9c0d1e2f3g4h5i6j7k8l9m0n1o2p3q4r5s6t7u8v9w0x1y2z3a4b5c6d7e8f9g0h1i2j3k4l5m6n7o8p9q0r1s2t3u4v5w6x7y8z9'
```

Comment:
SECRET_KEY should be kept secure .-. 

Store it in a env variable and call it when needed

Reply:
OH OH OH thank u..

Reply:
Ok noted on the gitignore! will edit